diagram.md

```diff
@@ -252,7 +265,7 @@ The `Attachments Element` is for attaching files to a Matroska file such as pict
 ```
 Figure: Representation of a `Attachments Element`.
 
-The `Tags Element` contains metadata that describes the `Segment` and potentially its `Tracks`, `Chapters`, and `Attachments`. Each Track or Chapter that those tags applies to has its UID listed in the tags. The Tags contain all extra information about the file, script writer, singer, actors, directors, titles, edition, price, dates, genre, comments, etc. And it allows you to enter many of these (title, edition, comments, etc.) in different languages.
+The `Tags Element` contains metadata that describes the `Segment` and potentially its `Tracks`, `Chapters`, and `Attachments`. Each `Track` or `Chapter` that those tags applies to has its UID listed in the `Tags`. The `Tags` contain all extra information about the file: scriptwriter, singer, actors, directors, titles, edition, price, dates, genre, comments, etc. Multiple tags (title, edition, comments, etc.) can be entered in different languages.
```

Comment:
> Multiple tags (title, edition, comments, etc.) can be entered in different languages.

I'm not happy with the wording "Multiple tags" here. Tags are key/value pairs. The intent is to be able to store multiple instances of the value portion for each key. Maybe something like this?

> Tags can contain their values in multiple languages. For example, a movie's "title" `Tag` might contain both the English original as well as the name it was released as in Germany.